cli/test/block-explorer-client.spec.ts

```diff
@@ -2,7 +2,7 @@ import { beforeEach, describe, expect, it } from "vitest";
 import { BlockExplorerClient } from "../src";
 import { ExternalApiError } from "../src/lib/errors";
 
-describe("BlockExplorerClient", () => {
+describe.skip("BlockExplorerClient", () => {
```

Comment:
Should this test suite be skipped?

Reply:
No. But they were doin an api call to etherscan and etherscan was down. But It's back online. Removing the sklip